src/internal/mediaUtil.ts

```diff
@@ -1,5 +1,7 @@
 import { AssembleAttachment, MediaChunk, MediaInputs, FileFormat, ImageUri } from '../public/media';
 
+export const fiftyMbInKb = 50 * 1024;
```

Comment:
Isn't this an ECS controlled item? If so, the ECS will not have any effect, if we gate this at JS SDK level by hard coding. I

Reply:
We are planning to introduce another function getMediaInChunks in JS library which will not assemble the media. The benefit will be we will be able to upload big videos from low end devices also. Developer doesn't need to assemble them, she can keeps uploading them to azure in chunks and get the url for viewing. getMedia and getMediaInChunks will use the exact same implementation underneath.